src/Jellyfin.LiveTv/Guide/GuideManager.cs

```diff
@@ -632,37 +615,35 @@ private async Task<LiveTvChannel> GetChannel(
         item.IndexNumber = info.EpisodeNumber;
         item.ParentIndexNumber = info.SeasonNumber;
 
-        forceUpdate = forceUpdate || UpdateImages(item, info);
+        forceUpdate = UpdateImages(item, info) || forceUpdate;
```

Comment:
This is a good fix, but it is not the same fix for Issue #13496 as PR #13497.  This one fixes the issue where _all_ images are not updated.

Reply:
```suggestion
        forceUpdate |= UpdateImages(item, info);
```

I think for clarity this should be updated to match the other pattern